osu.Game.Rulesets.Mania/UI/Components/ColumnHitObjectArea.cs

```diff
@@ -12,7 +11,7 @@ namespace osu.Game.Rulesets.Mania.UI.Components
 {
     public class ColumnHitObjectArea : HitObjectArea
     {
-        public readonly Container<HitExplosion> Explosions;
+        public readonly Container<Drawable> Explosions;
```

Comment:
```suggestion
        public readonly Container Explosions;
```

Same for other occurences.